packages/api/tsconfig.json

```diff
@@ -6,7 +6,9 @@
     "esModuleInterop": true,
     "lib": ["DOM", "DOM.Iterable", "ES2020"],
     "noImplicitAny": true,
-    "outDir": "dist/"
+    "outDir": "dist/",
+    "strict": true,
+    "useUnknownInCatchVariables": false
```

Comment:
Allows us to not have to do `catch (err: any)` nonsense that adds no value.

Reply:
Oh hell yeah I've been running into this a bunch in `rdme`, gonna steal this. Thanks!